sdk/storage/Azure.Storage.Blobs/tests/BlobTestBase.cs

```diff
@@ -246,75 +246,109 @@ public StorageSharedKeyCredential GetNewSharedKeyCredentials()
                     TestConfigDefault.AccountKey);
 
         public SasQueryParameters GetNewAccountSasCredentials(StorageSharedKeyCredential sharedKeyCredentials = default)
-            => new AccountSasBuilder
+        {
+            var builder = new AccountSasBuilder
             {
                 Protocol = SasProtocol.None,
                 Services = AccountSasServices.Blobs,
                 ResourceTypes = AccountSasResourceTypes.Container | AccountSasResourceTypes.Object,
                 StartsOn = Recording.UtcNow.AddHours(-1),
                 ExpiresOn = Recording.UtcNow.AddHours(+1),
-                Permissions = new BlobContainerSasPermissions { Read = true, Add = true, Create = true, Write = true, Delete = true, List = true }.ToString(),
                 IPRange = new SasIPRange(IPAddress.None, IPAddress.None)
-            }.ToSasQueryParameters(sharedKeyCredentials);
+            };
+            builder.SetPermissions(
+                AccountSasPermissions.Read |
```

Comment:
This was BlobContainerSasPermissions and now you've got AccountSasPermissions?

Reply:
It is using an AccountSasBuilder though, so I think it was incorrect before.